src/import/import.rs

```diff
@@ -54,7 +54,7 @@ impl<Client: ImportClient> ImportJob<Client> {
 
     pub fn run(&self) -> Result<()> {
         let start = Instant::now();
-        info!("{} start", self.tag);
+        info!("start"; "tag" => %self.tag);
```

Comment:
I think "job" is better than "tag" here.

Reply:
Shouldn't we keep consistent with other one:
https://github.com/tikv/tikv/blob/e78fb63e52069059484acd53c01b78b54f4cb029/src/raftstore/store/worker/read.rs#L113